### ISSUE TITLE
Make brew number unique

### ISSUE THREAD

**Issue by LondonRain** [Gremmel/kleiner-brauhelfer#85]
In one of the last two versions a brew number has been added. There is a sorting option for it in the brew overview tab. However numbers can be assigned at will.

I would like to make two proposals for a better user experience:

1.  They should be checked for uniqueness. The same number should not be assignable twice.
2.  If a new beer recipe is created or if in the brew number field the up arrow is clicked and the field was still 0 it shouldn't be set to 1 but to the next free available number.

**Comment by BourgeoisLab** [comment]
Sometimes I split my brew in two different tanks with two different yeasts. In this case I want both to have the same brew number. So I know that they belong together. So I vote against uniquness of the brew number.

I agree with point 2.

**Comment by LondonRain** [comment]
Good point. 10a/10b would be nice, but changing it from a numeric to a string field is probably is not worth it. Perhaps there could be a warning popup if I try to assign a new number which is already taken? This should not interfere with splitting, as the number does not change when splitting.
